sources/engine/Xenko.VirtualReality/Fove/FoveHmd.cs

```diff
@@ -13,7 +13,7 @@ namespace Xenko.VirtualReality
 {
     internal class FoveHmd : VRDevice
     {
-        private Texture nonSrgbFrame;
+        //private Texture nonSrgbFrame;
```

Comment:
Remove, don't comment out.

Reply:
OK as is, let's delete all related code in a separate pass